main/streams/plain_wrapper.c

```diff
@@ -125,11 +125,12 @@ typedef struct {
 	FILE *file;
 	int fd;					/* underlying file descriptor */
 	unsigned is_process_pipe:1;	/* use pclose instead of fclose */
-	unsigned is_pipe:1;			/* don't try and seek */
+	unsigned is_seekable:1;		/* don't try and seek, if not set */
 	unsigned cached_fstat:1;	/* sb is valid */
+	unsigned is_pipe:1;		/* stream is an actual pipe, currently Windows only*/
 	unsigned is_pipe_blocking:1; /* allow blocking read() on pipes, currently Windows only */
 	unsigned no_forced_fstat:1;  /* Use fstat cache even if forced */
-	unsigned _reserved:28;
+	unsigned _reserved:26;
 
```

Comment:
One member added, so it's 27?

Reply:
The whole size formerly has been 33 bits. I don't think that was intentional.